crypto/fipsmodule/cipher/e_aes.c

```diff
@@ -246,6 +246,7 @@ static int aes_cbc_cipher(EVP_CIPHER_CTX *ctx, uint8_t *out, const uint8_t *in,
     CRYPTO_cbc128_decrypt(in, out, len, &dat->ks.ks, ctx->iv, dat->block);
   }
 
```

Comment:
Why add service indicators in this file? These functions are only accessible through high-level functions, no?

Reply:
Moved the check to `int EVP_Cipher(EVP_CIPHER_CTX *ctx, uint8_t *out, const uint8_t *in, size_t in_len)` which directly accessed these static functions.